src/marks/contour.js

```diff
@@ -158,7 +158,12 @@ function contourGeometry({thresholds, interval, ...options}) {
         ? thresholds.range(...(([min, max]) => [thresholds.floor(min), max])(finiteExtent(VV)))
         : typeof thresholds === "function"
         ? thresholds(V, ...finiteExtent(VV))
-        : thresholds;
+        : typeof thresholds === "number"
+        ? ticks(...nice(...finiteExtent(VV), thresholds), thresholds)
```

Comment:
This isn’t sufficient. We’ll need these lines too (https://github.com/d3/d3-contour/pull/68):

https://github.com/d3/d3-contour/blob/0ba2dd047ce4cc2980025eff5c4fea989bf2da85/src/contours.js#L41-L42